lld/docs/DTLTO.rst

```diff
@@ -0,0 +1,60 @@
+Distributed ThinLTO (DTLTO)
+===========================
+
+DTLTO allows for the distribution of backend ThinLTO compilations via external
+distribution systems, e.g. Incredibuild. There is existing support for
+distributing ThinLTO compilations by using separate thin-link, backend
+compilation, and link steps coordinated by a build system that can handle the
+dynamic dependencies specified by the index files, such as Bazel. However, this
+often requires changes to the user's build process. DTLTO distribution is
+managed internally in LLD as part of the traditional link step and, therefore,
+should be usable via any build process that can support in-process ThinLTO.
+
+ELF LLD
+-------
+
+The command line interface for DTLTO is:
+
+- ``--thinlto-distributor=<path>``
+  Specifies the file to execute as a distributor process.
+  If specified, ThinLTO backend compilations will be distributed.
+
+- ``--thinlto-remote-opt-tool=<path>``
```

Comment:
Perhaps --thinlto-remote-compiler or --thinlto-remote-compile-tool. It's unlikely we will use the internal tool named `opt`

Reply:
See: https://github.com/llvm/llvm-project/pull/126654#discussion_r1968738956.

Reply:
Updated code to use `thinlto-remote-compiler`.